english/be-octodog/octodog.md

```diff
@@ -0,0 +1,3 @@
+This is the secret to become octodog. First you have to transformate to an octocat by complete this course then read at least 100 adventures.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.